### ISSUE TITLE
What is the value of N?

### ISSUE THREAD

**Issue by LoreBerli** [hkust-vgd/RFNet-4D#4]
I can't seem to be able to find the value of N := number of points per frame. 
* I see there are 17 frames for training and testing
* You use 512 randomly sampled points from the P_t point cloud to backprop

I would like to know what you're comparing the 10K point cloud **with** when you compute the Chamfer Distance at test time.
How many points go in, how many come out?

Thanks for any help.

**Comment by tuananh1007** [comment]
Hi LoreBerli,

Sorry for late reply as I'm having many deadlines. 

The value of N is 300 points which is same as other methods setting. You can find it [here](https://github.com/hkust-vgd/RFNet-4D/blob/613b431e1a306c4084f043a14c442bfcea806d29/configs/unflow.yaml#L5).

I don't get the point of "10K point cloud", could you elaborate it? 
As mentioned in the paper, we stated that "For calculating the losses during training, we randomly sampled a fixed number of 512 points in 3D space and time interval for reconstruction loss, and uniformly sampled trajectories of 100 points for flow estimation loss. More details can be found in our supplied code."

Hope it helps you. Please feel free to ask questions and I will try to answer it asap.

**Comment by LoreBerli** [comment]
Hi and thank you for the response.
I was referring to [this line](https://github.com/hkust-vgd/RFNet-4D/blob/613b431e1a306c4084f043a14c442bfcea806d29/eval.py#LL68C13-L68C26) but i probably misunderstood

**Comment by tuananh1007** [comment]
This issue is now closed as it has been resolved; however, you are welcome to reopen it if you have any further inquiries.
